simplq/src/styles/homePage.module.scss

```diff
@@ -101,11 +101,64 @@
   }
 }
 
-.slider {
-  @include text-color();
+
+.queue-info {
   @include center-vertically();
   @include center-horizontally();
-  margin: 2.5rem;
+  h2 {
+    @include standard-header();
+    font-family: $header-font;
+    font-size: 2rem;
+    margin: 1rem;
+  }
+  .queue-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+    padding: 0;
+    .queue {
+      flex: 0 1 400px;
+      text-align: left;
+      margin: 2rem;
```

Comment:
Can you stick replace px with rem everywhere in the SCSS files? We're sticking to rem everywhere else and keeping that consistent would be best.

Reply:
Okay, I'll do that. I see now that something is wrong with navigation should I try to fix it?

Reply:
If you can, then please go ahead. But, what's the problem?

Reply:
![image](https://user-images.githubusercontent.com/71724556/94994458-fdbaa400-0597-11eb-861e-364f0f80f950.png)
It looks like this on most phones.

Reply:
I think I've seen this before when you open a page, and then switch to mobile view. Try refreshing and see if it goes away.

If that's the case, a real user would not switch between desktop <-> mobile, and this is fine.

Reply:
Let's switch to `rem` and this is good to merge.

Reply:
I changed values but after refreshing the navbar hasn't changed.

Reply:
Let me merge your changes and see if I can reproduce this issue. 🤔